src/metabase/driver/clickhouse.clj

```diff
@@ -38,8 +38,8 @@
   (sql-jdbc.sync/pattern-based-database-type->base-type
    [[#"Array" :type/Array]
     [#"Bool" :type/Boolean]
-    [#"DateTime" :type/DateTime]
     [#"DateTime64" :type/DateTime]
+    [#"DateTime" :type/DateTime]
```

Comment:
Are there any tests for this change?

Reply:
should be covered by core tests of Metabase, but there is no particular impact in the match outcome anyway